setup-test-network.sh

```diff
@@ -171,10 +232,6 @@ rm docker-compose.override.yml
 timeout 30 bash -c 'until docker ps | grep indexer | grep -q healthy; do sleep 5; done'
 timeout 30 bash -c 'until docker ps | grep tap-agent | grep -q healthy; do sleep 5; done'
 
-# Mine some blocks
-# This is important for the gateway
-(./local-network/scripts/mine-block.sh 10) 2>/dev/null || true
-
```

Comment:
What was the story with taking this away, just out of curiosity?

Reply:
The previous version of local network required that(based on my experience not really a impl requirement) to "register" the smart contract deployment and indexer for the gateway.
Now it seems it is enough with the funding just before gateway deployment.